website/blog/2023-03-14-ferretdb-v-0-9-2.md

```diff
@@ -0,0 +1,117 @@
+
+## New Features
+
+In FerretDB v0.9.3, we’ve enabled the automatic creation of unique `_id` values for each document inserted into a collection.
+This new feature ensures that you don’t have duplicate `_id` values in the collection, preventing errors that could lead to the same `_id` values being accidentally inserted twice.
+
+In recent releases, we’ve worked on improving our support for aggregation pipeline stages and pushed more queries to the backend.
+FerretDB v0.9.3 improves on this, adding more stages and pushing down more queries, ensuring significantly better database performance.
+
+For instance, we’ve implemented `$sort` and `$group` aggregation pipeline stages.
+Suppose you have a collection named `order` that contains the following documents:
+
+```js
+{
+    _id: 1,
+    customer_name: "John Smith",
+    order_items: ["shirt", "pants", "shoes"],
+    order_total: 23.50
+},
+{
+    _id: 2,
+    customer_name: "Mary Johnson",
+    order_items: ["dress", "scarf"],
+    order_total: 35.20
+},
+{
+    _id: 3,
+    customer_name: "Bob Williams",
+    order_items: ["hat", "gloves", "socks"],
+    order_total: 10.99
+}
+```
+
+We can use the `$sort` aggregation pipeline stage to sort the documents in ascending or descending order based on a specific field.
+For example, to sort the documents in descending order based on the `order_total` field, set the value to `-1`:
+
+```js
+db.orders.aggregate([
+    { $sort: { order_total: -1 } }
+])
+```
+
+This will return the documents, sorted in descending order based on the `order_total` field.
+
+You can use the `$group` aggregation stage to group documents together.
+For example, the following query groups and counts the total number of documents in the `orders` collection.
+
+```js
+db.orders.aggregate([
+    {
+        $group: {
+            _id: null,
+            count: { $count: {} }
+        }
+    }
+])
+```
+
+As part of our mission to effectively enable more backends to run MongoDB workloads, basic aggregation pipeline support is now available for [Tigris](https://www.tigrisdata.com/).
+
+## Bug Fixes
+
+FerretDB v.0.9.3 has had several enhancements and bug fixes since the last release, indicating our commitment to building a reliable document database for our users.
+
+We’ve fixed dot notation errors for `$max`, `$min`, and `$mul` update operators.
+Previously, there were causing issues by failing to return errors when attempting to update paths that were not valid.
+The fixes now ensure that update operation will return an error when attempting to update an invalid path.
+
+We’ve also addressed issues with querying an embedded field in an array, especially with the `$pullAll` operator, which was preventing it from removing all instances from an array.
+One of our contributors [@yu-re-ka](https://github.com/yu-re-ka) helped fixed `saslStart` for particular clients, making it possible to achieve compatibility with [FastNetMon](https://fastnetmon.com/).
```

Comment:
We don't actually know yet if compatibility was achieved. It definitely did not work before, but it might also not work now. So it was _improved_, but not achieved

Reply:
PTAL